config.go

```diff
@@ -285,6 +308,12 @@ func pluginCmd(path string) *exec.Cmd {
 		}
 	}
 
+	// No plugin binary found, so try to use an internal plugin.
+	if strings.Contains(path, command.TFSPACE) {
+		parts := strings.Split(path, command.TFSPACE)
+		return exec.Command(parts[0], parts[1:]...)
+	}
```

Comment:
Just took a quick look -- this is easy to do for internal plugins but slightly more complicated for disk-based plugins because of the override behavior during discovery. I can still add this, though.